src/30_Advanced/Show_More_Button.html

```diff
@@ -0,0 +1,81 @@
+<!-- ## Show more button -->
+<!--
+This is a sample showing how to implement the "show more" design pattern.
+"Show more" is a very simple design pattern to navigate into pages with a lot of content,
+such e-commerce category pages, based on a simple user interaction on a button asking “Do you want to see more results?”
+
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+  <!-- ## Setup -->
+  <!-- Additionally used AMP components must be imported in the header. We use `amp-list` for showing a list of products -->
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+  <!-- We use `amp-bind` for dinamically changing the src of `amp-list` -->
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <!-- We use `amp-form` for making the call to get extra products after an user interaction -->
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <!-- We use `amp-mustache` for rendering the `amp-list` content -->
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+
+  <link rel="canonical" href="<%host%>/advanced/load_more_button/">
+  <title>Load more button</title>
+
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+
+</head>
+<body>
+  <!-- ## How to implement a show more button -->
+  <!-- You can implement a show more button by using `amp-list` and `amp-bind`, infact
+  `amp-list` data can be specified by using `amp-state` as a `src`. Here we are using 2 `amp-state`: `productsState`
+  represents the actual state of the list of shown products... -->
```

Comment:
s/represents/which contains/

Reply:
Can you mention that the amp-state uses the same json source as the amp-list

Reply:
done